shelf/bulk_update/utils.py

```diff
@@ -16,6 +15,40 @@ def run(args):
                 options provided to the update-search-index
                 script
     """
+    logger = set_up_logger("update-search-index", args)
+    config = get_config(args, logger.level)
+    bucket_list = get_bucket_list(args)
+    container = Container(config, logger)
+    runner = container.create_runner(update_search_index)
+    runner.run(bucket_list)
+
+
+def run_search_prune(args):
+    """
+        Runs search pruning based on given config file.
+
+        Args:
+            args(dict)
+    """
+    logger = set_up_logger("prune-search-index", args)
+    config = get_config(args, logger.level)
+    bucket_list = get_bucket_list(args)
+    container = Container(config, logger)
+    runner = container.create_runner(prune_search_index)
+    runner.run(bucket_list)
+
+
+def set_up_logger(task_name, args):
+    """
+        Sets up logging with the task name, and returns the logger.
+
+        Args:
+            task_name(basestring): The name of the task that's about to be run.
+            args(dict)
+
+        Returns:
+            RootLogger()
```

Comment:
I think you're right. I think I got `RootLogger` from going to the definition in my IDE. Fixed. cff638f